src/components/Card.jsx

```diff
@@ -8,7 +13,37 @@ export default function Card({ chatName, chatLink, from, to, dateFrom, dateTo, m
         redirectWindow.focus();
     }
 
-    console.log(dateFrom)
+    dayjs.extend(customParseFormat);
+    dayjs.extend(localizedFormat);
+    function formatDate(stringDate) {
```

Comment:
You can move it to bootstrap file and define as global function make locale as parameter and other logic there as well (those code lines that have dayjs imports)